.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+Implements [DHIS2-XXXX](https://dhis2.atlassian.net/browse/DHIS2-XXXX)
+
+### Description
+
+_text_
+
+---
+
+### Quality checklist
+
+Add _N/A_ to items that are not applicable and check them.
+
+<!--Checkmate-->
+
+-   [ ] Dashboard tested
+-   [ ] Cypress and/or Jest tests added/updated
+-   [ ] Docs added
+-   [ ] d2-ci dependency replaced (requires <https://github.com/dhis2/analytics/pull/XXX>)
+-   [ ] Tester approved (name)
+
+---
+
+### ToDos
+
+<!--Checkmate-->
+
+-   [ ] _todo_
+
+---
+
+### Known issues
+
+-   _issue_
+
+---
```

Comment:
Perhaps a bit pedantic, but I don't think "Known issues" should be checkboxes. When would you actually check the box? I've changed them to regular bullet points for now, because I think that's more apt, but happy to discuss of course.